rotkehlchen/exchanges/coinbase.py

```diff
@@ -684,6 +663,97 @@ def _process_normal_trade(self, event: dict[str, Any], trades: list[Trade]) -> N
 
         return None
 
+    def _process_normal_trade(self, event: dict[str, Any], timestamp: Timestamp) -> Trade | None:
+        """Turns a normal coinbase transaction into a rotki trade and returns it.
+
+        Sometimes the amounts can be negative which breaks rotki's logic which is why we use abs().
+
+        https://docs.cloud.coinbase.com/sign-in-with-coinbase/docs/api-transactions#transaction-types
+        If the coinbase transaction is not a trade related transaction nothing happens.
+        """
```

Comment:
can you add to the docstring the errors that those functions can raise? It is deserializationError, KeyError, UnknownAsset and UnsupportedAsset